src/bigunit.ts

```diff
@@ -286,15 +286,33 @@ export class BigUnit {
    * @returns BigUnit with the maximum value
    */
   public static max(unit1: BigUnit, unit2: BigUnit): BigUnit {
-    return unit1.gt(unit2) ? unit1 : unit2;
+    return this.compareUnits(unit1, unit2, (u1, u2) => u1.gt(u2));
```

Comment:
What is the advantage to this approach?

Reply:
Max and Min shared a few lines of code so this was just to reduce the duplication

Reply:
I don't think it's worth the added cognitive load and extra code